cmd/analyze/main.go

```diff
@@ -82,7 +83,7 @@ func dynamicAnalysis(manager *pkgecosystem.PkgManager, pkg *pkgecosystem.Pkg) {
 	sb := sandbox.New(manager.DynamicAnalysisImage(), sbOpts...)
 	defer cleanupSandbox(sb)
 
-	results, lastRunPhase, err := worker.RunDynamicAnalysis(sb, pkg)
+	results, fileWriteResults, lastRunPhase, err := worker.RunDynamicAnalysis(sb, pkg)
```

Comment:
can we generalize fileWriteResults a bit, and call it `rawAdditionalResults` or something similar?

Reply:
What if we make `dynamicanalysis.Result` into a top-level struct that contains all the data, rather than having to add a new struct to each function signature? Right now the only consumers of `dynamicanalysis.Result` are the bucket uploads, so if it's done in a way that keeps the data for different buckets in different fields of the new struct, then it should be okay.

I propose renaming the current `dynamicanalysis.Result` struct into `dynamicanalysis.StraceResult`, then changing `dynamicanalysis.Result` to look like 
```
type Result struct {
    Strace     StraceResult
    FileWrites []FileWriteResult
}
```
Then in the future, more fields can be added too.

Reply:
yep, that certainly makes things cleaner!